### PR TITLE
[15.0][ADD] account_move_search_line

### PR DESCRIPTION
**Pull request by AungKoKoLin1997**
This module provides to search with invoice line values in account invoice search views.

@qrtl

### CODE REVIEW

File: account_move_search_line/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+This module provides to search with invoice line values in account invoice search views.
```
Comment (yostashiro) [inline account_move_search_line/readme/DESCRIPTION.rst]:
```suggestion
This module allows for searching by invoice line values (labels) in the invoice search view.
```

**Review by yostashiro — APPROVED** [review]
Functional test in runboat.

**Review by rinaldifirdaus — APPROVED** [review]
Functional test: work as expected!

**Review by kanda999 — APPROVED** [review]
Function review; It works as expected in runboat.

**Comment by AungKoKoLin1997** [comment]
@OCA/accounting-maintainers Can we get this PR merged?

**Review by AaronHForgeFlow — APPROVED** [review]
Code review LG

**Pull request merged**